src/cve/utils/intel_source_score.py

```diff
@@ -0,0 +1,117 @@
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import json
+import logging
+import os
+
+from langchain_community.adapters.openai import ChatCompletion
+from langchain_community.chat_models import ChatOpenAI
+from langchain_core.messages import HumanMessage
+from langchain_nvidia_ai_endpoints import ChatNVIDIA
+
+from cve.data_models.config import RunConfig
+from cve.data_models.cve_intel import CveIntel
+from cve.utils import data_utils
+from cve.utils.loggers_factory import LoggingFactory
+from cve.utils.prompting import additional_intel_prompting
```

Comment:
@tmihalac  I Found  the root cause for the trace_id being omitted from logs, you need to change the import to relative path instead of absolute
```suggestion
from ..data_models.config import RunConfig
from ..data_models.cve_intel import CveIntel
from ..utils import data_utils
from ..utils.loggers_factory import LoggingFactory
from ..utils.prompting import additional_intel_prompting
```

Reply:
It didn't fix the problem

Reply:
It fixed the problem of the trace_id' value is absent from logs.